in-viewport.js

```diff
@@ -94,20 +93,35 @@ function createInViewport(container) {
   setInterval(debouncedCheck, 150);
 
   function isInViewport(elt, offset, cb) {
-    var visible = isVisible(elt, offset);
-    if (visible) {
-      if (cb) {
-        watching.splice(indexOf.call(watching, elt), 1);
-        cb(elt);
-      }
-      return true;
+    if (!cb) {
+      return isVisible(elt, offset);
+    }
+
+    var accessor = createAccessor(elt, offset, cb);
```

Comment:
I hesitate between `accessor`, `API` and `watcher`, because there is alreay a lot thing called `watch` or `watches`.
But you're right `watcher` is more explicit.
Perhaps we could rename `watches` and `createWatches`?